lib/cogctl/actions/bundles/create.ex

```diff
@@ -63,7 +63,7 @@ defmodule Cogctl.Actions.Bundles.Create do
         end
 
         commands = for command <- bundle.commands do
-          [command["name"], command["id"]]
+          [command.name, command.id]
```

Comment:
@christophermaier no it's not related to https://github.com/operable/cog-api-client/pull/120. That was another issue, though it was also related to bundles. I don't know the exact PR in cog_api that caused the problem.